internal/pkg/chainsync/internal/syncer/syncer_test.go

```diff
@@ -457,7 +460,7 @@ func TestSyncerStatus(t *testing.T) {
 
 	// advance the chain head, ensure status changes
 	t2 := builder.AppendOn(t1, 1)
-	require.NoError(t, syncer.HandleNewTipSet(ctx, block.NewChainInfo(peer.ID(""), "", t2.Key(), heightFromTip(t, t2)), false))
+	require.NoError(t, syncer.HandleNewTipSet(ctx, block.NewChainInfo(peer.ID(""), "", t2.Key(), heightFromTip(t, t2))))
```

Comment:
I find it a little disturbing that altering the `stageIfHeaviest` logic hasn't broken any tests. Does the fact that these tests still pass despite the fact we are not passing in the `trusted=true` flag indicate that the new code is exercised and working properly?

Reply:
I removed the status report on trustedness in order to get this to pass.  Trustedness of peers is a deprecated concept so I decided to throw it out here while doing other cleanup.

The stageIfHeaviest logic is adding in a hidden side effect to the syncer that is not yet consumed so it is expected that tests don't fail.